packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -630,103 +607,159 @@ class InterfaceProperty extends BaseProperty {
     protected readonly shouldEmitBody: boolean = false;
 }
 
-class TypedDict extends BasePythonClassType {
+class Struct extends BasePythonClassType {
+    protected directMembers = new Array<StructField>();
+
+    public addMember(member: PythonBase): void {
+        if (!(member instanceof StructField)) {
+            throw new Error('Must add TypedDictProperties to ValueType');
```

Comment:
I am assuming this is `StructField`, no?

Reply:
Oh yes